modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/asset/model/ObjectEntryAssetRenderer.java

```diff
@@ -103,7 +105,15 @@ public String getSummary(
 	@Override
 	public String getTitle(Locale locale) {
 		try {
-			return _objectEntry.getTitleValue();
+			String titleValue = _objectEntry.getTitleValue(
```

Comment:
Do you think it would be worthwhile to encapsulate this logic in a method in the ObjectEntryImpl class? That way, we avoid replicating this elsewhere.
I thought we could use the same pattern as `ObjectDefinitionModelImpl#getLabel(String languageId, boolean useDefault)`, so we would have a method `ObjectEntryImpl#getTitleValue(String languageId, boolean useDefault)`